tests/tools.vitruv.applications.pcmjava.tests/src/tools/vitruv/applications/pcmjava/tests/pcm2java/javahelper/FluentJavaClassBuilder.xtend

```diff
@@ -0,0 +1,237 @@
+package tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper
+
+import edu.kit.ipd.sdq.commons.util.java.Pair
+import java.util.ArrayList
+import java.util.List
+import org.eclipse.emf.ecore.util.EcoreUtil
+import org.emftext.language.java.classifiers.Class
+import org.emftext.language.java.containers.CompilationUnit
+import org.emftext.language.java.imports.Import
+import org.emftext.language.java.imports.ImportsFactory
+import org.emftext.language.java.members.ClassMethod
+import org.emftext.language.java.members.Constructor
+import org.emftext.language.java.members.Field
+import org.emftext.language.java.members.MembersFactory
+import org.emftext.language.java.modifiers.ModifiersFactory
+import org.emftext.language.java.parameters.ParametersFactory
+import org.emftext.language.java.references.ReferencesFactory
+import org.emftext.language.java.statements.StatementsFactory
+import org.emftext.language.java.types.TypeReference
+
+import static tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper.JavaCreatorsUtil.*
+
+import static extension edu.kit.ipd.sdq.commons.util.java.lang.IterableUtil.*
+
+/**
+ * Caution: the FluentJavaClassBuilder does not check for correctness of the builder API calls.
+ * This means that the caller has to ensure that references to fields (e.g. for getters, setters,
+ * initializations, ...) reference existing fields. Also the Builder doesn't prevent adding
+ * duplicated members (e.g. adding a getter for a field twice results in two identical getter methods).
+ */
+class FluentJavaClassBuilder{
```

Comment:
Maybe add a short comment what the class does before providing the caution hint.